setup.cfg

```diff
@@ -0,0 +1,4 @@
+[distutils]
+index-servers =
+  pypi
+
```

Comment:
We could also add the following information together:
```python
[zest.releaser]
create-wheel = yes
python-file-with-version = setup.py
```

Reply:
👍